openscenario/openscenario_interpreter/src/syntax/misc_object.cpp

```diff
@@ -35,8 +35,11 @@ MiscObject::operator traffic_simulator_msgs::msg::MiscObjectParameters() const
 {
   traffic_simulator_msgs::msg::MiscObjectParameters misc_object_parameters;
   {
-    misc_object_parameters.misc_object_category = boost::lexical_cast<String>(misc_object_category);
     misc_object_parameters.name = name;
+    /**
+     * @todo Currently, I use default value for misc object.
+     */
```

Comment:
Please indicate who the "I" is.
Also, even though this comment is a TODO comment, it does not indicate what needs to be done in the future.

What should be written here in the first place is not "using default values".
It is "WHY you use default values, WHY you think it is a problem, and WHAT you need to do in the future to fix it.

Reply:
I think this TODO comment is useless.
So, I deleted in this PR.
https://github.com/tier4/scenario_simulator_v2/pull/726/commits/30ee275090b1dfe2079c5b55ff014325844d079b